docs/Contributing/Adding-new-endpoints.md

```diff
@@ -219,20 +219,20 @@ This layer is tested in the integrations tests:
 Before we dive into specifics, let's discuss some context around the framework we are using and other tools that have
 been implemented.
 
-The main thing to be aware is that Fleet is using `go-kit` underneath. With it, we get all the concepts from the 
-framework such as decoders, transport, etc. We have decided `go-kit` is no longer the best framework for Fleet to use 
-anymore, but the cost of replacing it is higher than the cost of building some of the tooling we'll discuss here and 
+The main thing to be aware of is that Fleet uses `go-kit` underneath. With it, we get all the concepts from the 
+framework, such as decoders, transport, etc. We have decided `go-kit` is no longer the best framework for Fleet to use 
+anymore, but the cost of replacing it is higher than the cost of building some of the tools we'll discuss here and 
 maintaining it.
 
 The tools that were built were meant to abstract away layers that `go-kit` leaves available to its users in a way that 
 makes sense for our use case. 
 
 For instance, we found ourselves implementing extremely similar request decoding code. It varied very slightly from one 
-implementation to the next, and it varied in ways that Go wasn't capable of handling at the time. So we wrote a [generic
+implementation to the next, and it differed in ways that Go wasn't capable of handling at the time. So we wrote a [generic
```

Comment:
Should "Go" be in quotes here?

Reply:
I don't believe so, I haven't seen programming language names in quotes in docs before.